lpa-submissions-e2e-tests/cypress/support/commands.js

```diff
@@ -252,3 +246,33 @@ Cypress.Commands.add(
   'inputExtraConditionsExtraInformation',
   require('./appeals-extra-conditions/inputExtraConditionsExtraInformation'),
 );
+
+Cypress.Commands.add(
+  'goToDevelopmentPlanPage',
+  require('./appeals-questionnaire-navigation/goToDevelopmentPlanPage'),
+);
+
+Cypress.Commands.add(
+  'validateDevelopmentPlanPageTitle',
+  require('./appeals-development-plan/validateDevelopmentPlanPageTitle'),
+);
+
+Cypress.Commands.add(
+  'validateDevelopmentPlanPageHeading',
+  require('./appeals-development-plan/validateDevelopmentPlanPageHeading'),
+);
+
+Cypress.Commands.add(
+  'validateDevelopmentPlanErrorMessage',
+  require('./appeals-development-plan/validateDevelopmentPlanErrorMessage'),
+);
+
+Cypress.Commands.add(
+  'developmentPlanRadioButton',
+  require('./appeals-development-plan/developmentPlanRadioButton'),
+);
+
+Cypress.Commands.add(
+  'inputDevelopmentPlanDetails',
+  require('./appeals-development-plan/inputDevelopmentPlanDetails'),
+);
```

Comment:
Before this file gets out of hand (as it has done for FWA), it maybe worth considering extracting here:

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/e2e-tests/cypress/support/commands.js

Not essential by any means, just a thought / suggestion.

Maybe overkill also, idk anything about this side of the project.

Reply:
@shilpabasu22 Hi Shilpa can you advise? This is based on your style.

Reply:
@swdpcomputing I think Team 1 had hit the issue where command.js file had grown very big. We can edit/update our command.js file to mimic what Team1 has implemented. If you want, I can do that separate to this ticket as it doesn't impact the functionality that you have been working on.

Reply:
Sounds good and sensible. @a6software You happy with this?

Reply:
Yes, absolutely - both were only suggestions, all good from my pov. Cheers both!